src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.Scanner;
 public class Duke {
+    public static final String MARKDONE = "mark ";
+    public static final String MARKUNDONE = "unmark ";
+    public static final String TODO = "todo ";
+    public static final String DEADLINE = "deadline ";
+    public static final String EVENT = "event ";
+    public static final String LIST = "list";
+    public static final String BYE = "bye";
+    public static final String AT = "/at ";
+    public static final String BY = "/by ";
```

Comment:
The naming for these two is rather confusing as BYE at the above refers to the command while the AT here refers to the contains condition